src/app/pages/task-calendar/cloudsync/cloudsync-form/cloudsync-form.component.ts

```diff
@@ -412,13 +414,35 @@ export class CloudsyncFormComponent {
 
   protected providers: any;
   protected taskSchemas = ['encryption', 'fast_list', 'chunk_size', 'storage_class'];
+  public custActions: Array<any> = [
+    {
+      id : 'dry_run',
+      name : 'Dry Run',
```

Comment:
this needs a T(), also should be in the helptext file along with the rest of the text